ui/components-react/src/components-react/properties/renderers/PropertyView.scss

```diff
@@ -73,10 +73,15 @@ $text-font-color: var(--iui-color-text);
   .multiline {
     display: flex;
     align-items: baseline;
+
+    .content {
+      overflow: hidden;
+      text-overflow: ellipsis;
+    }
   }
 
   .expanded {
-    white-space: pre-wrap;
+    white-space: pre;
```

Comment:
Why was this changed to `pre`? How does it work with long string that has spaces but no new line characters?

Reply:
fixed